sos/report/plugins/ds.py

```diff
@@ -43,11 +44,20 @@ def setup(self):
             "/etc/dirsrv/admin-serv/password.conf"
         ])
 
+        # check for dsctl, and if it has the healthcheck option
+        HAVE_HEALTHCHECK = False
+        if os.path.exists('/usr/sbin/dsctl'):
```

Comment:
This is unnecessary - sos handles command failures automatically, especially for when commands do not exist.

Reply:
Ok, round 3, removed these checks, please review...